src/components/Tooltip/Tooltip.tsx

```diff
@@ -17,6 +17,8 @@ export interface TooltipProps {
   light?: boolean;
   /** Toggle whether the tooltip is visible */
   active?: boolean;
+  /** Remove overlay when cursor leaves Tooltip */
```

Comment:
This description is kind of misleading - what is the `Tooltip` vs the "overlay"? This prop is intended so that the `Tooltip` will be dismissed if the cursor enters what many would consider the "actual Tooltip". If we rename this prop, I think that will help clarify things, but we should also consider something like:

> Dismiss Tooltip once pointer is no longer over children